platform-sdk/swirlds-logging/src/main/java/com/swirlds/logging/external/benchmark/BenchmarkFactory.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.logging.external.benchmark;
+
+import com.swirlds.logging.api.extensions.handler.LogHandlerFactory;
+import com.swirlds.logging.console.ConsoleHandlerFactory;
+import com.swirlds.logging.file.FileHandlerFactory;
+import edu.umd.cs.findbugs.annotations.NonNull;
+
+/**
+ * Class that exposes the factories, so they can be used in the benchmarking module
+ */
+public class BenchmarkFactory {
```

Comment:
Can this class be moved to jmh src set?

Reply:
Oh, I remember; this class existed only for not having to individually export each package containing a factory to the benchmark when it was its module. Now, given that jmh source can access the files directly, I can do that or instantiate the factories without the need for this class.